pkg/server/testserver.go

```diff
@@ -645,6 +650,10 @@ func (ts *testServer) getSharedProcessDefaultTenantArgs() base.TestSharedProcess
 	if ts.params.Knobs.Server != nil {
 		args.Knobs.Server.(*TestingKnobs).DiagnosticsTestingKnobs = ts.params.Knobs.Server.(*TestingKnobs).DiagnosticsTestingKnobs
 		args.Knobs.LicenseTestingKnobs = ts.params.Knobs.LicenseTestingKnobs
+		args.Knobs.Server.(*TestingKnobs).ContextTestingKnobs.InjectedLatencyOracle =
+			ts.params.Knobs.Server.(*TestingKnobs).ContextTestingKnobs.InjectedLatencyOracle
+		args.Knobs.Server.(*TestingKnobs).ContextTestingKnobs.InjectedLatencyEnabled =
+			ts.params.Knobs.Server.(*TestingKnobs).ContextTestingKnobs.InjectedLatencyEnabled
```

Comment:
This can be de-duplicated, but I'm also thinking of some extra minor extra refactor in separate PR. I'll share the link here.

Reply:
I end up doing this in this PR only